vcd/datasource_vcd_network_isolated_v2.go

```diff
@@ -77,6 +77,29 @@ func datasourceVcdNetworkIsolatedV2() *schema.Resource {
 				Computed:    true,
 				Description: "Network prefix",
 			},
+			"dual_stack_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Boolean value if Dual-Stack mode is enabled",
+			},
+			"secondary_gateway": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
```

Comment:
```suggestion
```

Reply:
fixed all of them - thanks for seeing it - so odd tests didn't capture them